tfhe/src/c_api/high_level_api/config.rs

```diff
@@ -80,9 +80,10 @@ pub unsafe extern "C" fn config_builder_enable_custom_integers(
     catch_panic(|| {
         check_ptr_is_non_null_and_aligned(builder).unwrap();
 
+        let params: crate::shortint::ClassicPBSParameters = shortint_block_parameters.into();
```

Comment:
not generic enough to allow multi bit with the C API ?

Reply:
Its because of https://github.com/zama-ai/tfhe-rs-internal/issues/140